index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
Should we make the page title "Home" instead?

Reply:
I have fixed a problem here

https://github.com/snizhana-korolova/hello-world/pull/2/commits/e1073c70ad3eaa79c265cb23c3fbc0b3b0808921#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R7